superduperdb/container/document.py

```diff
@@ -31,6 +33,15 @@ def encode(self) -> t.Any:
         """Make a copy of the content with all the Encodables encoded"""
         return _encode(self.content)
 
+    def outputs(self, key: str, model: str) -> t.Any:
+        """
+        Get document ouputs on ``key`` from ``model``
+        :param key: Document key to get outputs from.
```

Comment:
Empty line after the first sentence, please!

Reply:
sure thanks